index.html

```diff
@@ -0,0 +1,247 @@
+                    product , feature or website Look through some of my
+                    work and experience! If you like what you see and have
+                    a project you need coded, don’t hestiate to contact me.
+                </p>
+                <h3 class="intro2">LET’S CONNECT</h3>
+                <ul class="social">
+                    <li class="social-icon">
+                        <img src="images/github.png" alt="github">
+                    </li>
+                    <li class="social-icon">
+                        <img src="images/Linkedin.png" alt="Linkedln">
+                    </li>
+                    <li class="social-icon">
+                        <img src="images/fingers.png" alt="two fingers">
+                    </li>
+                    <li class="social-icon">
+                        <img src="images/twitter.png" alt="twitter">
+                    </li>
+
+                    <li class="social-icon">
+                        <img src="images/m.png" alt="m">
+                    </li>
+                </ul>
+                <div class="button-resume">
+                    <button type="button" class="btn">
+                        Get my resume
+                    </button>
+                </div>
+            </div>
+            <div class="right">
+                <div class="lang">
+
+                </div>
+                <ul class="language-list">
+                    <li class="lang">
+                        <p>
+                            Languages
+                        </p>
+                        <img class="imglang" src="images/arrowdown.png"
+                            alt="arrow">
+                    </li>
+                    <li class="lang-skill">
+                        <img src="images/js.png" alt="js">
+                        <p>
+                            JavaScript
+                        </p>
+                    </li>
+                    <li class="lang-skill">
+                        <img src="images/css.png" alt="html">
+                        <p>
+                            CSS
+                        </p>
+                    </li>
+                    <li class="lang-skill">
+                        <img src="images/html.png" alt="css">
+                        <p>
+                            HTML
+                        </p>
+                    </li>
+                    <li class="lang">
+                        <p>
+                            FrameWorks
+                        </p>
+                        <img class="imglang" src="images/arrowside.png" alt="">
+                    </li>
+                    <li class="line"></li>
+                    <li class="lang">
+                        <p>
+                            Skills
+                        </p>
+                        <img class="imglang" src="images/arrowside.png" alt="">
+                    </li>
+                    <li class="line"></li>
+                </ul>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be in a list `ul` and each of them a `li`.

Kindly fix this.